### ISSUE TITLE
Questions regarding mobile support prioritization; should mobile support be required for launch?

### ISSUE THREAD

**Issue by davisshaver** [WordPress/gutenberg#7779]
In the most recent [update on Gutenberg](https://wordpress.org/news/2018/07/update-on-gutenberg/), a couple mentions of mobile support:

> In Progress: Support on Automattic's iOS and Android Apps
> August and beyond: There will be a mobile version of Gutenberg by the end of the year.

Integrating a self-hosted site with the Automattic apps requires Jetpack.

As a WordPress.org project, is Gutenberg prioritizing Automattic's needs over the community's? Here is how Jetpack describes the importance of mobile:

> Mobile devices are the ~~future~~ present.
> 
> If your site is not optimized for mobile devices or you’re not able to manage it from your phone or tablet you’re behind the curve.
> 
> Mobile browsing is constantly on the rise and you yourself are probably relying on your phone or tablet more and more.

I have written a number of posts using Gutenberg on iOS Chrome and I think that the experience is quite good. It doesn't seem out of scope to polish the experience such that users experience parity, or near parity, when they first begin using Gutenberg. The distinction between desktop and mobile makes sense to developers, but I think users expect parity – especially because that's what they have today in the Classic Editor.

**Comment by davisshaver** [comment]
@karmatosed Thanks for responding. But I am not clear what meaning I should takeaway from this sentence:
> There will be a mobile version of Gutenberg by the end of the year.

What does this statement mean?

**Comment by karmatosed** [comment]
@davisshaver the mobile apps are a different thing and that is what it was referring to, a mobile version isn't the same as working on mobile. Having a truly mobile experience is different from adapting. That's one reason the apps which are found here: https://github.com/wordpress-mobile exists. You can follow along the progress here: https://github.com/wordpress-mobile and also in each repo here (depending on platform):

- https://github.com/wordpress-mobile/WordPress-Android
- https://github.com/wordpress-mobile/WordPress-iOS

**Comment by davisshaver** [comment]
Thank you @karmatosed, that makes sense. I read that sentence to mean that mobile web support would be slated for post-5.0. It sounds like actually that is referring to improved native app support, rather than saying that mobile web support would come after the first release.

**Comment by vsajewel** [comment]
So I take the above to mean that Gutenberg is supposed to work when writers are using mobile devices (similar to how the class editor does.) I first tried it a couple of weeks ago and it worked pretty well except for some problems I had with captions in image galleries.

But I’ve been trying to use it the last two days for creating a new post and editing existing one and it seems like nothing is working properly...rendering it unusable. 

The main problem is that you can’t position blocks. The arrows to move a block's position  don't appear. You can’t place an image within the body of your content. These can only appear at the beginning or the end of a post. 

Last is that it freezes up for no reason that I can surmise. I'm attaching a screenshot for one instance of that.
![8736b6b0-7d89-42bb-bbd6-214c60a8500b](https://user-images.githubusercontent.com/19415439/50129021-fd271100-023c-11e9-9a96-38ffc7076dba.jpeg)

**Comment by davisshaver** [comment]
@vsajewel Are these problems specific to mobile? The screenshot is of desktop. I have run into issues like this too but it wasn't clear that they were mobile specific.

**Comment by vsajewel** [comment]
Yes...specific to mobile...at least as far as I know. I haven't tried using a computer because I don’t have a working one currently...and never really do. I do all my work on iPads. The screenshot was taken on an iPad...perhaps in Desktop mode. 

Rereading your original entry I realized that you suggest Chrome  might offer more stability. I'll try that. 

In regards to the larger discussion above. The reason I chose and continue to use WordPress is because of it's high degree of mobile friendliness. I've not found another editing Suite that approaches the breadth of mobile friendliness that WordPress offers...I use WordPress.com.

When I began 4 years ago I only used iPads...and still do today.The classic editor and the new one have always worked exceptionally well for me. I've used both  as well as the Aztec editor for ios. That too works exceptionally well now, but it’s had ups and downs...more than likely because I  use the beta version. 

The reason I use all 3 is because each offers a few features that the others don’t. For example I can only use Custom Sizes for images when I use the older Classic editor. I can also change the setting for what an image links to in that editor (and only that editor) and up until recently, when that feature was also added to the Aztec editor for ios. 

Overall the oldest classic editor provides for the greatest level of fine tuning...but having to scroll up to the top and then back down to add things like heading's sizes or links can be frustrating when a post is extremely long as most of mine are. 

That's why I like the newer 'Classic' editor better for general writing...the floating toolbar is awesome!

But recently, say within the last 3 months, reliability of both classic editors has degraded so significantly that they are now barely usable. My personal theory has been that slowly they are being purposefully degraded to drive greater user acceptance of the new Gutenberg version. 
 
The one thing that worries me about the Gutenberg editor is the emphasis on drag and drop. Because ios devices really can’t drag and drop things very consistently. So far my fears haven't been warranted as the small arrows for moving around blocks of content have worked well up until the last couple of days when the small arrows disappeared. When the arrow are present and functioning correctly they make dragging and dropping a nonissue. 

I was unsure of where to post my original question about that problem...so I choose to post it here. I’m open to suggestions if there's a better place for it. Thanks for your comment davisshaver!
